source/containers.tex

```diff
@@ -17053,16 +17053,14 @@
 
 \pnum
 The effect of calling a constructor
-that takes
-both \tcode{key_container_type} and \tcode{mapped_container_type} arguments with
-containers of different sizes is undefined.
+that takes both \tcode{key_container_type} and \tcode{mapped_container_type} arguments
+with containers of different sizes
+is undefined.
 
 \pnum
 The effect of calling a constructor or member function
-that takes a \tcode{sorted_unique_t} argument with
-a container, containers, or range
-that is not sorted with respect to \tcode{key_comp()}, or
-that contains equal elements,
+that takes a \tcode{sorted_unique_t} argument
+with a container, containers, or range that is not sorted with respect to \tcode{key_comp()}, or that contains equal elements,
```

Comment:
```suggestion
with a container, containers, or range that is not sorted with respect to \tcode{key_comp()},
or that contains equal elements,
```

Reply:
Bikeshed: To keep the line length under some amount? In a vacuum I'd want to keep "with _these arguments_" all on one line...

Reply:
Yes, to keep the line shorter, and following https://sembr.org/ to do so.